community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -484,7 +484,7 @@ object projects:
 
   lazy val catsEffect3 = SbtCommunityProject(
     project        = "cats-effect-3",
-    sbtTestCommand = "ciJVM",
+    sbtTestCommand = "ciJVM; ciJS",
```

Comment:
To help prevent JS-specific regressions e.g. https://github.com/lampepfl/dotty/issues/15494.

Reply:
Let's see how much slowdown this causes.

Reply:
Ugh, we can't even run these for annoying reasons (namely: an npm dependency). I don't think there's a clean workaround unless the idea in https://github.com/scala-js/scala-js-js-envs/issues/20#issuecomment-1025130006 lands which would allow us to drop the npm dep.

Reply:
```suggestion
    sbtTestCommand = "ciJVM",
```